### PR TITLE
Better 'no posts to be found' message

### PR DESCRIPTION
**Pull request by Sjmarf**
Centered the loading graphic and replace "no posts to be found" message (see images).
<p float="left">

</p>
<p float="left">

</p>

### CODE REVIEW

**Comment by Sjmarf** [comment]
Added the fancy error-message from `ExpandedPostView` too

**Comment by Sjmarf** [comment]
In future we could do something similar for the inbox tab, offering a button to show all messages if the user is only viewing unread messages

File: Mlem/Views/Tabs/Feeds/Components/NoPostsView.swift [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                        .aspectRatio(contentMode: .fit)
+                        .frame(width: unreadItems == 0 ? 35 : 50)
+                        .padding(.bottom, unreadItems == 0 ? 8: 12)
+                    Text(title)
+                    
+                    if unreadItems != 0 {
+                        Text(
+                            (unreadItems == 1 ? "1 available post was" : "\(unreadItems) available posts were")
+                            + " hidden because you have 'hide read' enabled."
+                        )
```
Comment (EricBAndrews) [inline Mlem/Views/Tabs/Feeds/Components/NoPostsView.swift:35]: Could be more concise:

`\(unreadItems) read post \(unreadItems == 1 ? " has" : "s have") been hidden`

Along with the "show read," I think it's clear enough what's going on

**Review by EricBAndrews — CHANGES_REQUESTED** [review]
Nice! Minor wording comment, but fantastic work as always

**Review by EricBAndrews — APPROVED** [review]

**Pull request merged**